src/linter/dotLibrary/DotLibraryLinter.ts

```diff
@@ -0,0 +1,65 @@
+import {LintMessageSeverity} from "../LinterContext.js";
+import LinterContext from "../LinterContext.js";
+import {deprecatedLibraries} from "../../utils/deprecations.js";
+import {SaxEventType, Tag as SaxTag} from "sax-wasm";
+import {parseXML} from "../../utils/xmlParser.js";
+import {ReadStream} from "node:fs";
+
+export default class DotLibraryLinter {
+	#contentStream;
+	#resourcePath;
+	#context: LinterContext;
+
+	constructor(resourcePath: string, contentStream: ReadStream, context: LinterContext) {
+		this.#contentStream = contentStream;
+		this.#resourcePath = resourcePath;
+		this.#context = context;
+	}
+
+	async lint() {
+		try {
+			const dotLibraryDependencyTags = await this.#parseDotLibrary(this.#contentStream);
+			this.#analyzeDeprecatedLibs(dotLibraryDependencyTags);
+		} catch (err) {
+			const message = err instanceof Error ? err.message : String(err);
+			this.#context.addLintingMessage(this.#resourcePath, {
+				severity: LintMessageSeverity.Error,
+				message,
+				ruleId: "ui5-linter-parsing-error",
```

Comment:
Please use the `formatMessage` util
as it's done here: https://github.com/SAP/ui5-linter/pull/197/files#diff-47ae50dc9ac33799e2dd8a28629230419b8dcb6e7c0492f17801a53f6b59d7ceR96

Reply:
Resolved by [8f7284a](https://github.com/SAP/ui5-linter/pull/104/commits/8f7284a9fbdd0db544bbe5550c95b02a91e6113f)

Reply:
Resolved?